os/mm/mm_heap/mm_heapinfo_parse_heap.c

```diff
@@ -182,6 +215,26 @@ void heapinfo_parse_heap(FAR struct mm_heap_s *heap, int mode, pid_t pid)
 					heap_dbg("0x%x | %8d |   %c    |            |       |\n", node, node->size, 'F');
 				}
 			}
+#ifdef CONFIG_WATCHDOG
```

Comment:
same as previous

Reply:
This is difficult to seperate other function because it has to be output in the middle of outputting the log.